lib/factory_bot_rails/generator.rb

```diff
@@ -19,12 +15,10 @@ def run
     def generator
       if factory_bot_disabled?
         Generators::NullGenerator
+      elsif test_framework == :rspec
+        Generators::RSpecGenerator
       else
-        if test_framework == :rspec
-          Generators::RSpecGenerator
-        else
-          Generators::NonRSpecGenerator
-        end
+        Generators::NonRSpecGenerator
```

Comment:
tbh I kind of like the previously `if/else` because the `elsif` is on first glance unrelated to to factory bot being disabled... does that make sense?

Anyway this is just preference, but figured I'd bring it up in case 💅

Reply:
I changed because RuboCop was yelling about nested conditionals. I will change it back, since it is not related to this PR at all.